docs/guides/performance/join_operations.md

```diff
@@ -3,9 +3,33 @@ layout: docu
 title: Join Operations
 ---
 
-## Forcing a Join Order
+## How to Force a Join Order
 
 DuckDB has a cost-based query optimizer, which uses statistics in the base tables (stored in a DuckDB database or Parquet files) to estimate the cardinality of operations.
+
+### Turn off the Join Order Optimizer
+
+```sql
+SET disabled_optimizers = 'join_order';
```

Comment:
Potentially, users would also want to disable left/right swapping for joins with `'build_side_probe_side'`, then they will build a left-deep tree by chaining `JOIN` clauses. They can create a bushy plan by using parentheses.

Reply:
Thanks! I worked this into the PR.